src/CMakeLists.txt

```diff
@@ -49,61 +41,24 @@ ADD_GEN_QRC(LMMS_RCC_OUT lmms.qrc
 	"${CONTRIBUTORS}"
 )
 
-# Paths relative to lmms executable
-FILE(RELATIVE_PATH LIB_DIR_RELATIVE "/${BIN_DIR}" "/${LIB_DIR}")
-FILE(RELATIVE_PATH PLUGIN_DIR_RELATIVE "/${BIN_DIR}" "/${PLUGIN_DIR}")
-ADD_DEFINITIONS(-DLIB_DIR="${LIB_DIR_RELATIVE}" -DPLUGIN_DIR="${PLUGIN_DIR_RELATIVE}" ${PULSEAUDIO_DEFINITIONS} ${PORTAUDIO_DEFINITIONS})
-INCLUDE_DIRECTORIES(
-	${JACK_INCLUDE_DIRS}
-	${SAMPLERATE_INCLUDE_DIRS}
-	${SNDFILE_INCLUDE_DIRS}
-	${SNDIO_INCLUDE_DIRS}
-	${FFTW3F_INCLUDE_DIRS}
-)
-
-IF(NOT ("${SDL2_INCLUDE_DIR}" STREQUAL ""))
-	INCLUDE_DIRECTORIES("${SDL2_INCLUDE_DIR}")
-ELSEIF(NOT ("${SDL_INCLUDE_DIR}" STREQUAL ""))
-	INCLUDE_DIRECTORIES("${SDL_INCLUDE_DIR}")
-ENDIF()
-
-IF(LMMS_HAVE_WEAKJACK)
-	LIST(APPEND LMMS_SRCS "${WEAKJACK_INCLUDE_DIRS}/weak_libjack.c")
-	LIST(APPEND LMMS_INCLUDES "${WEAKJACK_INCLUDE_DIRS}/weak_libjack.h")
-	INCLUDE_DIRECTORIES("${WEAKJACK_INCLUDE_DIRS}")
-	ADD_DEFINITIONS(-DUSE_WEAK_JACK=1 -DNO_JACK_METADATA=1)
-ENDIF()
-
-IF(NOT ("${PORTAUDIO_INCLUDE_DIR}" STREQUAL ""))
-	INCLUDE_DIRECTORIES("${PORTAUDIO_INCLUDE_DIR}")
-ENDIF()
-
-IF(NOT ("${PULSEAUDIO_INCLUDE_DIR}" STREQUAL ""))
-	INCLUDE_DIRECTORIES("${PULSEAUDIO_INCLUDE_DIR}")
-ENDIF()
-
-IF(NOT ("${OGGVORBIS_INCLUDE_DIR}" STREQUAL ""))
-	INCLUDE_DIRECTORIES("${OGGVORBIS_INCLUDE_DIR}")
-ENDIF()
-
-IF(NOT ("${LAME_INCLUDE_DIRS}" STREQUAL ""))
-	INCLUDE_DIRECTORIES("${LAME_INCLUDE_DIRS}")
-ENDIF()
-
-LIST(APPEND LMMS_SRCS "${RINGBUFFER_DIR}/src/lib/ringbuffer.cpp")
-
 # Use libraries in non-standard directories (e.g., another version of Qt)
 IF(LMMS_BUILD_LINUX)
 	LINK_LIBRARIES(-Wl,--enable-new-dtags)
 	SET(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
 ENDIF()
 SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 
+message("My waeck jack include dir is ${LMMS_HAVE_WEAKJACK} ${WEAKJACK_INCLUDE_DIR}")
```

Comment:
@lukas-w Is this intended to be committed?

Reply:
@PhysSong Just for the purpose of debugging CI build failures. But it appears they pass now :) How was Debian fixed?